src/components/Filter/FilterBar.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import ToggleButton from '@mui/material/ToggleButton';
+import { Tooltip, Typography } from '@mui/material';
+import { Grid } from '@mui/material';
+import SupportIcon from '../SupportIcon/SupportIcon';
+import { withStyles } from '@mui/styles';
+
+export interface FilterBarProps {
+  filter: Array<Filter>;
+  updateFilter: (newFilters: Array<Filter>) => void;
+  supporting: number;
+  balanced: number;
+  disputing: number;
+}
+
+const CustomToggleButton = withStyles({
+  root: {
+    padding: 0,
+    paddingBottom: 4,
+    borderTopLeftRadius: 0,
+    borderTopRightRadius: 0,
+    width: 120,
+    height: 75,
+    '& .MuiTypography-root': {
+      fontFamily: 'DM Mono',
+    },
+  },
+})(ToggleButton);
+
+const FilterBar: React.FC<FilterBarProps> = ({
```

Comment:
Can you write a short FilterBar.stories.tsx?

Reply:
I don't know how, I don't know how to handle this:
<img width="405" alt="image" src="https://user-images.githubusercontent.com/38763304/187098369-c5dbeded-1327-410a-8589-eb0fd70c9e55.png">